src/components/pages/Landing/Landing.js

```diff
@@ -1,10 +1,22 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import 'antd/dist/antd.css';
-import { Link } from 'react-router-dom';
+import { useHistory } from 'react-router';
 import { Card } from 'antd';
 import { Button } from 'antd';
 
 const Landing = () => {
+  const match = useHistory();
+
+  useEffect(() => {
+    if (JSON.parse(localStorage.getItem('consent')) === true) {
```

Comment:
Only thing I am worried about is what is the odds of someone coming to the website with a session token called consent already set to true?